src/components/UserPortal/OrganizationCard/OrganizationCard.tsx

```diff
@@ -121,7 +121,6 @@ function organizationCard(props: InterfaceOrganizationCardProps): JSX.Element {
       }
       refetch();
     } catch (error: unknown) {
-      /* istanbul ignore next */
       if (error instanceof Error) {
```

Comment:
1. Make this file use CSS exclusively from `src/style/app.module.css`, with the exception of packaged based CSS
1. Delete the CSS file it currently uses.
1. This will help with future attempts to having Dark Mode
1. Use a variable from `src/style/app.module.css` .
1. If this is the sidebar color, name it `--side-bar` , or whatever this section of code is defining. 

- https://github.com/PalisadoesFoundation/talawa-admin/issues/3296